docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst

```diff
@@ -86,6 +86,55 @@ to discuss reasonable assumptions with subject matter experts to determine the
 most appropriate measure to which to apply the GBD risk effects in our vivarium
 simulations.
 
+In GBD 2021, some continuous risk exposures were modeled with general
+relationships between the exposure level and the relative risk (going
+beyond the log-linear relationship assumed for previous iterations).
+Interpreting the GBD estimates is straightforward, once you have
+chased down all of the necessary definitions.  The relevant estimates
+include a column for exposure level, as well as columns for 500 draws
```

Comment:
```suggestion
include a column for point exposure level, as well as columns for 500 draws
```

Reply:
what do you mean by point?

Reply:
May not be the right word. I originally thought that this would be categorical, i.e. there would be _ranges_ of exposure. But it seems there are just exact values of exposure, which you interpolate.